rest/data/project_settings.go

```diff
@@ -45,11 +77,29 @@ func (sc *DBConnector) CopyProject(ctx context.Context, projectToCopy *model.Pro
 	}
 	// set the same admin roles from the old project on the newly copied project.
 	if err := sc.UpdateAdminRoles(projectToCopy, projectToCopy.Admins, nil); err != nil {
-		return nil, errors.Wrapf(err, "Database error updating admins for project '%s'", newProject)
+		return nil, errors.Wrapf(err, "Database error updating admins for project '%s'", opts.NewProjectIdentifier)
 	}
 	return apiProjectRef, nil
 }
 
+func (sc *DBConnector) verifyUniqueProject(name string) error {
+	_, err := sc.FindProjectById(name, false)
+	if err == nil {
```

Comment:
we should probably double check that the error is a StatusNotFound error, because the error can also come from FindMergedProjectRef.

Reply:
even with FindBranchProjectRef, we may get an error from here: https://github.com/evergreen-ci/evergreen/blob/main/db/query.go#L78. 
err won't be nil so we won't return.